.github/workflows/release-defaults.yml

```
@@ -0,0 +1,33 @@
+name: Upload files to the GitHub release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    name: Publish ${{ matrix.file_name }}
+    runs-on: ${{ matrix.file_name }}
```

Shouldn't this be `matrix.os`?

```suggestion
    runs-on: ${{ matrix.os }}
```